src/components/editor-page/sidebar/export-markdown-sidebar-entry.tsx

```diff
@@ -5,18 +5,22 @@
  */
 
 import React, { useCallback } from 'react'
+import sanitize from 'sanitize-filename'
 import { Trans, useTranslation } from 'react-i18next'
 import { useNoteMarkdownContent } from '../../../hooks/common/use-note-markdown-content'
 import { download } from '../../common/download/download'
 import { SidebarButton } from './sidebar-button'
+import { useSelector } from 'react-redux'
+import { ApplicationState } from '../../../redux'
 
 export const ExportMarkdownSidebarEntry: React.FC = () => {
-  useTranslation()
-
+  const { t } = useTranslation()
+  const noteTitle = useSelector((state: ApplicationState) => state.noteDetails.noteTitle)
   const markdownContent = useNoteMarkdownContent()
   const onClick = useCallback(() => {
-    download(markdownContent, `title.md`, 'text/markdown') //todo: replace hard coded title with redux
-  }, [markdownContent])
+    const sanitized = sanitize(noteTitle)
```

Comment:
You don't need the `useSelector` hook here. The hook is only relevant, if the component should rerender or call a side effect if the redux store changes. Here you can grab the value directly from the redux store by importing the redux store and using the current value.

example code 

`import { store } from './redux'`

`return store.getState().apiUrl.apiUrl`

Reply:
Done

Reply:
Almost. Read the value in the callback.

Reply:
Fixed